app/controllers/evaluations_controller.rb

```diff
@@ -58,36 +57,23 @@ def new
   def edit
     @evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find_by(id: params[:id])
```

Comment:
this could also use the find(id) method to raise NotFound if the record doesn't exist
```suggestion
    @evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find(params[:id])
```

Reply:
I think it can technically use the `set_evaluation_and_submission_assignment` even though no params will be assigned it should still work ok if you want to do that instead, a little less code and more reuse

Reply:
I tried this but I think it errors out on the assign_attributes in the init service since the params don't have the required :evaluations key in the params. Probably some future adjustments to make to how those services/routes are setup

Reply:
ok yeah, that makes sense